### DISCUSSION TITLE
zh-cn/protocol/integration

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#170] [Comments]
# zh-cn/protocol/integration

集成文档 接口介绍 AsstAppendTask 接口原型 接口说明 添加任务 返回值 AsstTaskId 若添加成功，返回该任务 ID, 可用于后续设置任务参数； 若添加失败，返回 0 参数说明 AsstHandle handle 实例句柄 const char* type 任务类型 const char* params 任务参数，json str...

https://docs.maa.plus/zh-cn/protocol/integration.html

**Comment by warningme** [comment]
刷理智还缺少备选关卡的配置项

**Reply by ABA2396** [reply]
备选由 UI 自主实现，core 无相关参数

**Comment by Starseeker-xz** [comment]
Fight的stage关卡获取API改成 https://api.maa.plus/MaaAssistantArknights/api/gui/StageActivityV2.json 了吧？

**Reply by ABA2396** [reply]
是
